src/state/data/slice.ts

```diff
@@ -1,21 +1,22 @@
 import { createApi } from '@reduxjs/toolkit/query/react'
-import { gql } from 'graphql-request'
+import { gql, GraphQLClient } from 'graphql-request'
 import { FeeAmount } from '@uniswap/v3-sdk'
 import { reduce } from 'lodash'
+import { graphqlRequestBaseQuery } from '@rtk-query/graphql-request-base-query'
 
-import { graphqlBaseQuery, UNISWAP_V3_GRAPH_URL } from './common'
 import { FeeTierDistribution, PoolTVL } from './types'
 
-export const dataApi = createApi({
+export const UNISWAP_V3_GRAPH_URL = 'https://api.thegraph.com/subgraphs/name/ianlapham/uniswap-v3-alt'
```

Comment:
one note is we should be using this endpoint 

https://thegraph.com/explorer/subgraph/uniswap/uniswap-v3

Reply:
ah, thanks!